src/backend/utils/wiki-feed-parser.js

```diff
@@ -50,7 +50,7 @@ module.exports.parseData = function() {
             feed.push(`${line}`);
             let obj = {
               name: feed[feed.length - 1],
-              link: feed[feed.length - 2],
+              url: feed[feed.length - 2],
```

Comment:
Will this change break anything else we have?  It's probably fine, just want to confirm that we don't have any callers who depend on this yet.

Reply:
It shouldn't really, although I noticed that Travis is failing on something, but it looks like the problem is with environment variables, not sure why though.

Reply:
Travis failure is #387, unrelated to your changes.  OK, we should be good.  Get another review, and let's merge this!